source/_components/sensor.sense.markdown

```diff
@@ -30,11 +30,12 @@ sensor:
     - daily_production
 ```
 
-Two types of sensors can be monitored will be created with the following names:
+Two types of sensors can be monitored and will be created with the following names:
 - **Active Usage/Production**: Current active power usage/production in Watts. Updated every 30 seconds.
 - **Daily Usage/Production**: Daily power usage/production in kWh. Updated every 5 minutes.
+- ...
 
-Weekly, Monthly, and Yearly variants are also available.
```

Comment:
Not a fan of the Oxford comma?

Reply:
Haha, we don't have this in German^^
If this is your innermost wish you can add it again ;p